extensions/spring-data-jpa/deployment/src/test/java/io/quarkus/spring/data/deployment/LoginEvent.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.spring.data.deployment;
+
+import java.time.ZonedDateTime;
+
+import javax.persistence.*;
```

Comment:
Please remove the star import

Reply:
Sorry, I can't figure out what part of Intellj does this again, now that I setup the Eclipse formatter. I'll fix it.

Reply:
No problem :)